syncd/FlexCounter.cpp

```diff
@@ -1478,7 +1478,9 @@ void FlexCounter::runPlugins(
     {
         std::to_string(COUNTERS_DB), // TODO
         COUNTERS_TABLE,
-        std::to_string(m_pollInterval * 1000)
+        std::to_string(m_pollInterval * 1000),
+        std::to_string(ASIC_DB),
```

Comment:
also why you need ASIC_DB and CONFIG_DB ? oncly counters_db should be utilized here

Reply:
is this hard requirement? I'm utilising asic db to get port speed. I see that pfc scripts do the same.

Reply:
hard requirement it to not use hardcoded databases indexes/names and use those which are loaded from config file, also why do you need port speed here?

Reply:
It's for calculating port utilization, port byte rate is divided by port speed.

Reply:
@kcudnik Decided to avoid passing multiple db ids to the lua plugins and use only counters DB.
Removed hardcoded db ids.